components/voting.tsx

```diff
@@ -117,6 +131,37 @@ export default class Voting extends React.PureComponent<VotingProps, VotingState
     return []
   }
 
+  async submit() {
+    const vote = {
+      Id: this.props.voteId,
+      Indices: this.state.votes.map(
+        id => this.props.sessions.indexOf(this.props.sessions.filter(s => s.Id === id)[0]) + 1,
```

Comment:
Minor: `find ` would be a little cleaner that `filter` seeing as we are just interested in the first item.